### PR TITLE
analytics v2

### PR DESCRIPTION
**Pull request by david-goodfellow**

### CODE REVIEW

**Comment by jsani-radar** [comment]
Can we get a clubhouse / screenshot / launch date included here?

Want to see how the page looks before it goes

**Comment by david-goodfellow** [comment]
image
image

**Comment by david-goodfellow** [comment]
https://app.clubhouse.io/radarlabs/story/6765/analytics-v2-documentation

File: docs/trip-tracking.mdx [hunk 1]
```diff
@@ -125,28 +125,57 @@ Or, listen for [trip events](/trip-tracking#trip-events) sent to a webhook. From
...
+
 Trip stats apply to the environment in which you view the metrics. For example, when viewing the metrics in the test environment, the metrics only include trip data from the test environment.
 
+### Statistic cards
+
+- **`Median wait time`**: the median duration of time that it takes between the moment when a user arrives at a destination (`user.arrived_at_trip_destination`) and the moment when the trip is marked complete (`completed`). This statistics omits trips which are missing an arrival timestamp.
+- **`Number of trips`**: the total count of all trips for a specific project which is live and has a status of `completed`.
+- **`Median trip time`**: the median duration of time that it takes between the moment when the user starts a trip (`user.started_trip`) and the moment when they arrive at the destination (`user.arrived_at_trip_destination`). This statistics omits trips which are missing an arrival timestamp.
+- **`Repeat customers`**: the total number of your users who have completed more than one trip.
```
Comment (aidancleary) [inline docs/trip-tracking.mdx]: Shouldn't this read:
Repeat customers: the total number of users who have completed more than one trip. Like the other statistic cards, the number of repeat customers is affected by the filters set in the Filters section.

File: docs/trip-tracking.mdx [hunk 2]
```diff
@@ -125,28 +125,57 @@ Or, listen for [trip events](/trip-tracking#trip-events) sent to a webhook. From
...
-
-### Calculations
-
-Wait time and trip time are only calculated for trips that have a status of `completed` and have timestamp entries for `started_at`, `arrived_at`, and `completed_at`. These calculations exclude expired trips.
+Radar offers analytics on trip data and location opt-in rates from users in your project. [Analytics](https://radar.io/dashboard/analytics) on trip data can be found on the Analytics page and grouped under the Trips tab. Analytics on location permission opt-in rates is grouped under the Permissions tab. Read more about Permissions [here](https://radar.io/documentation/faqs#location-permissions).
 
 ### How to filter your data
 
-**Date Range:** Select a date from the drop-down date picker. Make sure that the date in the left-hand box falls before the date in the right-hand box.
+The Filters button at the top of the page offers you ways to look at more granular views of your trip data. The filters set with this button apply to the statistics cards and graphs as well as the data in the By Destination table.
```
Comment (aidancleary) [inline docs/trip-tracking.mdx:132]: delete "you", so it just says "offers ways to look at more.."

File: docs/trip-tracking.mdx [hunk 3]
```diff
@@ -125,28 +125,57 @@ Or, listen for [trip events](/trip-tracking#trip-events) sent to a webhook. From
...
 
-**Radar User ID**: This field requires the unique Radar-generated ID and does not accept "external ID" or "device ID" in its place. You can find the unique Radar-generated ID for each user by navigating to the Users page and clicking "View" in the corresponding row of the user.
+**Date Range:** Select a date from the drop-down date picker. Make sure that the date in the left-hand box falls before the date in the right-hand box. (The default date range when first accessing the Analytics page encompasses all trip data in your project, meaning that you'll see the dates of your earliest trips.)
```
Comment (aidancleary) [inline docs/trip-tracking.mdx:134]: I think change this to:

> Date Range: A date range can be selected from the drop-down date picker. Make sure that the date in the left-hand box falls before the date in the right-hand box. When accessing the Analytics page, the default date range encompasses all trip data in your project, meaning that you'll see a default range from the date of your earliest trip to yesterday. All trip analytics are updated nightly and trip analytics from today will not be shown in the Analytics page until tomorrow.

**Review by aidancleary — CHANGES_REQUESTED** [review]
made 3 small edits

File: docs/trip-tracking.mdx [hunk 4]
```diff
@@ -125,28 +125,57 @@ Or, listen for [trip events](/trip-tracking#trip-events) sent to a webhook. From
...
 
 **Destination Geofence External ID**: Each geofence has an external ID. You can find the external ID for a specific geofence in the "External ID" column on the Geofences page in the sidebar, or you can navigate to the Trips page and click on the external ID listed under "Destination". This field can be used alone or in tandem with the Destination Geofence Tag field.
 
+**Travel Mode:** Select a specific mode of travel to filter all trip data in the Analytics view to only a specific mode. If there is no mode selected, all trip data, regardless of travel mode, will be shown.
+
 Trip stats apply to the environment in which you view the metrics. For example, when viewing the metrics in the test environment, the metrics only include trip data from the test environment.
 
+### Statistic cards
```
Comment (jsani-radar) [inline docs/trip-tracking.mdx:144]: let's come up with a better header here, this is gonna be on the right sidebar - maybe "Trip metrics" or just "Metrics"?

File: docs/trip-tracking.mdx [hunk 5]
```diff
@@ -125,28 +125,57 @@ Or, listen for [trip events](/trip-tracking#trip-events) sent to a webhook. From
...
-- **`Number of trips`**: the total count of all trips for a specific project which is live and has a status of `completed`.
-- **`Median trip time`**: the median duration of time that it takes between the moment when the user starts a trip (`user.started_trip`) and the moment when they arrive at the destination (`user.arrived_at_trip_destination`).
-
-### Calculations
-
-Wait time and trip time are only calculated for trips that have a status of `completed` and have timestamp entries for `started_at`, `arrived_at`, and `completed_at`. These calculations exclude expired trips.
+Radar offers analytics on trip data and location opt-in rates from users in your project. [Analytics](https://radar.io/dashboard/analytics) on trip data can be found on the Analytics page and grouped under the Trips tab. Analytics on location permission opt-in rates is grouped under the Permissions tab. Read more about Permissions [here](https://radar.io/documentation/faqs#location-permissions).
```
Comment (jsani-radar) [inline docs/trip-tracking.mdx:128]: **for** users in your project. Let's hyperlink users with https://radar.io/documentation/api#users as well

File: docs/trip-tracking.mdx (hunk 2 above)
Comment (jsani-radar) [reply docs/trip-tracking.mdx:132]: Remove "set with this button" so the second sentence reads "The filters will apply to the statistics cards..."

File: docs/trip-tracking.mdx (hunk 3 above)
Comment (jsani-radar) [reply docs/trip-tracking.mdx:134]: do we have validation around the left hand / right hand constraint? fine if not, but feels like a common pattern, which would reduce a sentence here

File: docs/trip-tracking.mdx [hunk 6]
```diff
@@ -125,28 +125,57 @@ Or, listen for [trip events](/trip-tracking#trip-events) sent to a webhook. From
...
 
 **Destination Geofence External ID**: Each geofence has an external ID. You can find the external ID for a specific geofence in the "External ID" column on the Geofences page in the sidebar, or you can navigate to the Trips page and click on the external ID listed under "Destination". This field can be used alone or in tandem with the Destination Geofence Tag field.
 
+**Travel Mode:** Select a specific mode of travel to filter all trip data in the Analytics view to only a specific mode. If there is no mode selected, all trip data, regardless of travel mode, will be shown.
+
 Trip stats apply to the environment in which you view the metrics. For example, when viewing the metrics in the test environment, the metrics only include trip data from the test environment.
 
+### Statistic cards
+
+- **`Median wait time`**: the median duration of time that it takes between the moment when a user arrives at a destination (`user.arrived_at_trip_destination`) and the moment when the trip is marked complete (`completed`). This statistics omits trips which are missing an arrival timestamp.
```
Comment (jsani-radar) [inline docs/trip-tracking.mdx:146]: singular: `statistic`, or `metric` depending on previous convo

File: docs/trip-tracking.mdx [hunk 7]
```diff
@@ -125,28 +125,57 @@ Or, listen for [trip events](/trip-tracking#trip-events) sent to a webhook. From
...
+- **`Number of trips`**: the total count of all trips for a specific project which is live and has a status of `completed`.
```
Comment (jsani-radar) [inline docs/trip-tracking.mdx:147]: instead of "which is live" can we we put "live" in between "all trips"

File: docs/trip-tracking.mdx [hunk 8]
```diff
@@ -125,28 +125,57 @@ Or, listen for [trip events](/trip-tracking#trip-events) sent to a webhook. From
...
+- **`Median trip time`**: the median duration of time that it takes between the moment when the user starts a trip (`user.started_trip`) and the moment when they arrive at the destination (`user.arrived_at_trip_destination`). This statistics omits trips which are missing an arrival timestamp.
```
Comment (jsani-radar) [inline docs/trip-tracking.mdx:148]: remove "of the time that it takes" so it reads "the median duration between"

File: docs/trip-tracking.mdx [hunk 9]
```diff
@@ -125,28 +125,57 @@ Or, listen for [trip events](/trip-tracking#trip-events) sent to a webhook. From
...
+- **`Repeat customers`**: the total number of your users who have completed more than one trip. (like the other statistic cards, the number of repeat customers is affected by the filters set in the Filters section)
```
Comment (jsani-radar) [inline docs/trip-tracking.mdx:149]: "total number of users", do we need the parenthesis statement here?

File: docs/trip-tracking.mdx [hunk 10]
```diff
@@ -125,28 +125,57 @@ Or, listen for [trip events](/trip-tracking#trip-events) sent to a webhook. From
...
+
+The graphs for each card show the change over time in each statistic. The time windows of each graph are automatically determined by the dates set in the Filters button. To see data grouped by hour, set the date range to encompass 2 days or fewer. To see data grouped by day, set the date range to encompass 21 days or fewer. To see data grouped by weeks, set the date range to encompass 60 days or fewer. To see data grouped by month, the date range must encompass more than 60 days of data.
```
Comment (jsani-radar) [inline docs/trip-tracking.mdx:151]: should we have a chart to see the correspondence between # of days / grouping? feels like that'd be easier to grok

Last ___ Days | Grouping
2                     | Hour
21                   | Day
60                  | Week
60+                | Month

File: docs/trip-tracking.mdx [hunk 11]
```diff
@@ -125,28 +125,57 @@ Or, listen for [trip events](/trip-tracking#trip-events) sent to a webhook. From
...
+- **`Repeat customers`**: the total number of your users who have completed more than one trip. (like the other statistic cards, the number of repeat customers is affected by the filters set in the Filters section)
+
+The graphs for each card show the change over time in each statistic. The time windows of each graph are automatically determined by the dates set in the Filters button. To see data grouped by hour, set the date range to encompass 2 days or fewer. To see data grouped by day, set the date range to encompass 21 days or fewer. To see data grouped by weeks, set the date range to encompass 60 days or fewer. To see data grouped by month, the date range must encompass more than 60 days of data. 
+
+### By Destination table
+
+The By Destination table groups your trip data by individual destination geofence and allows you to identify your slowest and fastest stores by wait time as well as your most popular stores by the number of trips made to the store.
```
Comment (jsani-radar) [inline docs/trip-tracking.mdx:155]: can we bold "slowest and fastest stores" / "popular stores"

File: docs/trip-tracking.mdx [hunk 12]
```diff
@@ -125,28 +125,57 @@ Or, listen for [trip events](/trip-tracking#trip-events) sent to a webhook. From
...
+
+By default, the By Destination table is sorted alphabetically based on the Destination column. Clicking on any column header will sort the table by that column's metrics in ascending order. To sort the column in descending order, click again on the column header.
```
Comment (jsani-radar) [inline docs/trip-tracking.mdx:157]: To toggle the sort order of the column, click the column header.

**Review by jsani-radar — COMMENTED** [review]
looking solid overall, may seem like a lot of comments but mostly just want to condense language to streamline things / make it easier to scan

File: docs/trip-tracking.mdx (hunk 4 above)
Comment (jsani-radar) [inline docs/trip-tracking.mdx:144]: let's come up with a better header here, this is gonna be on the right sidebar - maybe "Trip metrics" or just "Metrics"?

File: docs/trip-tracking.mdx (hunk 9 above)
Comment (david-goodfellow) [reply docs/trip-tracking.mdx:149]: defer to @aidancleary

File: docs/trip-tracking.mdx (hunk 1 above)
Comment (david-goodfellow) [reply docs/trip-tracking.mdx:149]: Took out the parenthesis based on @jsani-radar [comment](https://github.com/radarlabs/documentation/pull/30#discussion_r677057573)

File: docs/trip-tracking.mdx (hunk 4 above)
Comment (david-goodfellow) [reply docs/trip-tracking.mdx]: @aidancleary made the switch to trip metrics but want your input

File: docs/trip-tracking.mdx [hunk 13]
```diff
@@ -125,28 +125,64 @@ Or, listen for [trip events](/trip-tracking#trip-events) sent to a webhook. From
...
+- **`Number of trips`**: the total count of all live trips for a specific project which have a status of `completed`.
+- **`Median trip time`**: the median duration between the moment when the user starts a trip (`user.started_trip`) and the moment when they arrive at the destination (`user.arrived_at_trip_destination`). This statistics omits trips which are missing an arrival timestamp.
+- **`Repeat customers`**: the total number of your users who have completed more than one trip.
+
+The graphs for each card show the change over time in each statistic. The time windows of each graph are automatically determined by the dates set in the Filters button. To see data grouped by hour, set the date range to encompass 2 days or fewer. To see data grouped by day, set the date range to encompass 21 days or fewer. To see data grouped by weeks, set the date range to encompass 60 days or fewer. To see data grouped by month, the date range must encompass more than 60 days of data. 
+
+| Last ___ Days  | Time Grouping |
```
Comment (jsani-radar) [inline docs/trip-tracking.mdx:153]: let's get an updated screenshot? I'm thinking the ___ won't look too great, maybe "Number of Days" as the column header?

File: docs/trip-tracking.mdx [hunk 14]
```diff
@@ -125,28 +125,64 @@ Or, listen for [trip events](/trip-tracking#trip-events) sent to a webhook. From
...
+| ------------- | ------------- |
+| 2 | Hour |
+| 21 | Day |
+| 60 | Week |
+| 60+ | Month |
+
+### By Destination table
+
+The By Destination table groups your trip data by individual destination geofence and allows you to identify your **slowest and fastest stores** by wait time as well as your most popular stores by the number of trips made to the store.
```
Comment (jsani-radar) [inline docs/trip-tracking.mdx:162]: bold "most popular stores"

File: docs/trip-tracking.mdx (hunk 10 above)
Comment (jsani-radar) [reply docs/trip-tracking.mdx:151]: think we can remove the last 4 sentences since we now have this table

**Review by jsani-radar — APPROVED** [review]
few small nits, overall looks good - let's confirm "Trip metrics" as the top-level header before merging this in

**Comment by david-goodfellow** [comment]
image

File: docs/trip-tracking.mdx (hunk 9 above)
Comment (aidancleary) [reply docs/trip-tracking.mdx]: Ah, i wrote "This statistics omits trips..." , need to delete the last S in statistics for both 'median wait time' and 'median trip time'

File: docs/trip-tracking.mdx (hunk 4 above)
Comment (aidancleary) [reply docs/trip-tracking.mdx]: we should call it "Trips Overview" since that's what the header is for the section here

**Comment by aidancleary** [comment]
on line 142 can we change "Trip stats" to "Trip analytics"

**Pull request merged**